File: android/src/main/kotlin/com/snowplowanalytics/snowplow_flutter_tracker/readers/messages/SetDomainUserIdMessageReader.kt
package com.snowplowanalytics.snowplow_flutter_tracker.readers.messages

class SetDomainUserIdMessageReader(val values: Map<String, Any>) {
    private val valuesDefault = values.withDefault { null }

    val tracker: String by values
    val domainUserId: String? by valuesDefault
}

File: ios/Classes/readers/messages/SetDomainUserIdMessageReader.swift
//
//  SetDomainUserIdMessageReader.swift
//  snowplow_flutter_tracker
//
//  Created by Matus Tomlein on 04/01/2022.
//

import Foundation

struct SetDomainUserIdMessageReader: Decodable {
    let tracker: String
    let domainUserId: String?
}
